.github/workflows/tests.yml

```diff
@@ -39,12 +39,12 @@ jobs:
       - name: Install root test dependencies
         run: |
           cd tests/fixtures/root
-          composer install
+          rm -rf vendor && composer install
```

Comment:
Is this is because GH actions was caching old vendor resources, or because something is being saved to the repo that needs to be replaced?

Reply:
I was seeing some issues with composer not able to symlink but I think they've worked themself out. Going to remove!